workbench_utils.py

```diff
@@ -6189,7 +6189,7 @@ def download_file_from_drupal(config, node_id):
                 else:
                     continue
     else:
-        logging.error(f'Attempt to fetch media list {media_list_url} returned an {r.status_code} HTTP response.')
+        logging.error(f'Attempt to fetch media list {media_list_url} returned an {media_list_response.status_code} HTTP response.')
```

Comment:
Yes, thanks for catching that. I'll need to test with the updates variable name before merging though.

Reply:
Thank for catching that one @ysuarez. I can fix it off-PR if you want and credit you, or you can update this PR by removing the other proposed fix, if you prefer. Let me know!